client/wfnews-war/src/main/angular/src/app/components/full-details/evac-alert-full-details/evac-alert-full-details.component.html

```diff
@@ -0,0 +1,116 @@
+<div class="title-panel">
+  <div class="full-details-title">Evac Alert for {{evacData.name}}</div>
+  <div class="alert-panel"><img height="20" width="20" class="icon" src="/assets/images/svg-icons/evacuation_alert.svg"
+      alt="evacuation_alert" />Alert</div>
+  <p class="info">A legal order that prohibits access to certain areas in order to limit the risk of fire, address a
+    public safety concern or avoid interference with fire control.</p>
+  <button (click)="navToMap()" mat-button class="map-button"><img height="20" width="20" class="icon"
+      src="/assets/images/svg-icons/map.svg" alt="Map" />View on Map</button>
+</div>
+<hr>
+<div class="info-container">
+  <div class="info"><img height="20" width="20" class="icon" src="/assets/images/svg-icons/carbon_calendar.svg"
+      alt="Calendar" />Issued on {{convertToDateTime(evacData.issuedDate)}}</div>
+  <div class="info"><img height="20" width="20" class="icon" src="/assets/images/svg-icons/location-pin.svg"
+      alt="Location Pin" />Issued by {{evacData.issuingAgency}}</div>
+  <div class="info"><img height="20" width="20" class="icon" src="/assets/images/svg-icons/carbon_layers.svg"
+      alt="Layers" />Fire Evacuation Orders and Alerts Layer</div>
+</div>
+<hr>
+<div class="info-container" *ngIf="incident">
+  <div class="full-details-secondary-title">Associated Wildfire</div>
+  <div class="fire-info-container">
+    <div class="wildfire-title">{{incident.incidentName}}<img height="20" width="20" class="wildfire-title-icon"
```

Comment:
might want to clean up these 20, 20. And I see there is no wildfire-title-icon class in the files

Reply:
Thanks I fixed them now